client/packages/common/src/ui/components/navigation/AppNavLink/StyleFreeLink.tsx

```diff
@@ -0,0 +1,8 @@
+import Link, { LinkProps } from '@mui/material/Link';
```

Comment:
This component could be added to the StatsPanel if there isn't a good chance it would be used elsewhere

Reply:
Do we want to be using the Material `Link` component, or just the react-router-dom one?  (as used in `Breadcrumbs.tsx`). Or should we just import from `common`?

@mark-prins ?